docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,201 @@
+
+* **Type:** :ref:`prop-type-string`
+
+A comma-separated list of regexes for the names of the cipher algorithms to use.
+
+``http-client.log.compression-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Enable log file compression. The client uses the ``.gz``` format for log files.
+
+``http-client.log.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``false``
+
+Enable HTTP requests logging in the ``http-client.log.path``.
+
+``http-client.log.flush-interval``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``10s``
+
+Frequency of flushing the log data to disk.
+
+``http-client.log.max-history``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``15``
+
+Retention limit of log files in days. Files older than the ``max-history`` are
+deleted when the HTTP client creates files for new logging periods.
+
+``http-client.log.max-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-data-size`
+* **Default value:** ``1GB``
+
+Maximum total size of all log files on disk.
+
+``http-client.log.path``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``var/log/``
+
+Sets the path of the log files. All log files are named ``http-client.log``, and
+have the prefix of the specific HTTP client added. For example,
+``jwk-http-client.log``.
+
+``http-client.log.queue-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``10000``
+* **Minimum value:** ``1``
+
+Size of the HTTP client logging queue.
+
+TLS and security properties
+---------------------------
+
+``http-client.http-proxy.secure``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``false``
+
+Enable HTTPS for the proxy.
+
+``http-client.https.hostname-verification``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Verify that the server hostname matches the server DNS name in the
+SubjectAlternativeName (SAN) field of the certificate.
+
+``http-client.key-store-password``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+
+Password for the keystore.
+
+``http-client.key-store-path``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+
+File path on the server to the keystore file.
+
+``http-client.secure-random-algorithm``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+
+Set the secure random algorithm for the connection. The default varies by
+operating system. Algorithms are specified according to standard algorithm name
+documentation.
+
+Possible types include ``NativePRNG``, ``NativePRNGBlocking``,
+``NativePRNGNonBlocking``, ``PKCS11`` ``SHA1PRNG``, and ``Windows-PRNG``.
```

Comment:
Missing comma. Also, probably remove `Windows-PRNG` since we don't run on Windows